### ISSUE TITLE
[Package Request]: StevenMayall.MusicBee

### ISSUE THREAD

**Issue by KrisSchneider** [microsoft/winget-pkgs#65008]
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [ ] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [X] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Steven Mayall
Publisher Url: https://www.getmusicbee.com/
Package Name: StevenMayall.MusicBee
Description: MusicBee makes it easy to manage, find, and play music files on your computer. MusicBee also supports podcasts, web radio stations and SoundCloud integration
Package Version: 3.4
Installer URL: https://files1.majorgeeks.com/10afebdbffcd4742c81a3cb0f6ce4092156b4375/multimedia/MusicBeeSetup_3_4_Update2.zip

Current Package Identifier (for new versions): 
https://www.majorgeeks.com/files/details/musicbee.html
https://files1.majorgeeks.com/10afebdbffcd4742c81a3cb0f6ce4092156b4375/multimedia/MusicBeeSetup_3_4_Update2.zip

**Comment by KrisSchneider** [comment]
I think there should be a different package from the package already offered in MS Store. Not everyone likes the MS Store.

**Comment by Trenly** [comment]
This is blocked by microsoft/winget-cli#140
![72E8309F-F1DE-4903-AB0E-BE026B1CF44B](https://user-images.githubusercontent.com/12611259/177023784-3883b30e-3d2a-48dc-b411-4ba4e3a23e6e.jpeg)

**Comment by KrisSchneider** [comment]
The executable file is in a zip file though.

**Comment by KrisSchneider** [comment]
Is it impossible for Winget to extract a .zip archive's content into a new folder, and then open the .exe file inside the newly created folder?

**Comment by Trenly** [comment]
> Is it impossible for Winget to extract a .zip archive's content into a new folder, and then open the .exe file inside the newly created folder?

At the monent, yes; But, the work for that feature is planned

**Comment by KrisSchneider** [comment]
Oh, ok. I have a suggestion, has the winget team tried implementing the 7zip CLI into the program for archive extracting?

**Comment by vedantmgoyal2009** [comment]
You can see https://github.com/microsoft/winget-cli/pull/2270/files, to know how the feature will be implemented in WinGet.

**Comment by KrisSchneider** [comment]
> You can see https://github.com/microsoft/winget-cli/pull/2270/files, to know how the feature will be implemented in WinGet.

Oh, that's great to see. Thanks for showing me.

**Comment by dpprdan** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/issues/4216

**Comment by KrisSchneider** [comment]
> Duplicate of #4216

Should I close this issue?

**Comment by Trenly** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/issues/4216

**Comment by Trenly** [comment]
Duplicate of #4216
